### DISCUSSION TITLE
[Question]: bonding box color

### DISCUSSION THREAD

**Discussion by kei81131** [blakeblackshear/frigate#12369] [Ask A Question] [answered]
### What is your question:

I am interested in the meaning of bonding boxes color.
May I know if there is any docs? 
Thank you.

**Comment by hawkeye217** [comment, marked as answer]
On snapshots and MQTT messages, the colors are selected from a predefined color palette and don't have any explicit meaning. If you're watching the debug view, a gray bounding box means your object is stationary (0.14 beta and later only) and a blue bounding box means your object was recently lost.

See the code here: https://github.com/blakeblackshear/frigate/blob/d2f88491b14c27249190e0c8e74a7ace50db6abf/frigate/object_processing.py#L505

**Reply by kei81131** [reply]
oh, cause after i updated dev-d2f8849
I see the captured bbox color changed from yellow to lightblue
I mistakenly though each color should has a specific meaning 

thanks to clear my mind

**Reply by hawkeye217** [reply]
The code for the color palette was recently updated, which is why you saw the change.
